be/src/exec/vectorized/json_scanner.cpp

```diff
@@ -398,11 +406,11 @@ void JsonReader::_construct_column(const rapidjson::Value& objectValue, Column*
         break;
     }
     case rapidjson::Type::kFalseType: {
-        column->append_strings(std::vector<Slice>{Slice("0")});
+        column->append_strings(literal_0_slice_vector);
         break;
     }
     case rapidjson::Type::kTrueType: {
-        column->append_strings(std::vector<Slice>{Slice("1")});
+        column->append_strings(literal_0_slice_vector);
```

Comment:
literal_1_slice_vector is define and not to use.
maybe here should be literal_1_slice_vector?

Reply:
Yes, will fix